src/kbmod/search/stack_search.cpp

```diff
@@ -280,7 +294,8 @@ std::vector<float> StackSearch::create_curves(Trajectory t, const std::vector<Ra
         /* Do not use get_pixel_interp(), because results from create_curves must
          * be able to recover the same likelihoods as the ones reported by the
          * gpu search.*/
-        float pix_val = imgs[i].get_pixel(t.x + int(times[i] * t.vx + 0.5), t.y + int(times[i] * t.vy + 0.5));
+        Point p({t.y + times[i] * t.vy + 0.5f, t.x + times[i] * t.vx + 0.5f});
```

Comment:
Right. Best guess here is that this used to be a part of a for loop or something and the someone noticed rounding is sometimes funky, added 0.5 in. This 0.5 is now "extra" because the context was that origin was in each pixel or something like that. We should either shortcut this directly make sure `Trajectory` returns `Point` and `Point.to_index` does the correct rounding.